internal/server/v1/observations/bulk_point_linked.go

```diff
@@ -47,7 +47,7 @@ func BulkPointLinked(
 		return nil, status.Errorf(codes.InvalidArgument,
 			"Missing required argument: linked_entity")
 	}
-	if linkedProperty == "" {
+	if linkedProperty != "containedInPlace" {
```

Comment:
So we only support 'containedInPlace' as linkedProperty for now... Shall we document this somewhere?

Or, shall we indicate this in the error message below? Something like:

"Invalid argument: linked_property. Now the following values are supported: containedInPlace, XXX, YYY, etc."

Reply:
Done